test/Go/testdata/go.mod.edgecases

```diff
@@ -20,6 +20,8 @@ replace repo/B => alias/repo/B v0.1.0
 replace (
 	repo/C => alias/repo/C v0.0.0-20180207000608-000000000003
 	repo/E => alias/repo/E v0.0.0-20170808103936-000000000005+incompatible
+    foo v0 => ../foo
+    bar => /foo/bar/baz
```

Comment:
[nit] Are the differences in leading whitespace significant?  If not, can we normalize them?

Reply:
it was completely unintentional, but they're accidentally significant insofar as proving our parser is whitespace-insensitive -- so I guess I'll keep them